Mathlib/Data/Int/DivMod.lean

```diff
@@ -13,6 +13,10 @@ import Mathlib.Init
 
 namespace Int
 
+theorem natCast_emod (m n : Nat) : (↑(m % n) : Int) = m % n := ofNat_emod m n
```

Comment:
I presume these names are used in core, and this PR adds transitionary aliasses? Would you like to put an adaptation note, so we remember to remove those again? (Or will that automatically happen? Then I don't mind either way.)

Reply:
Yeah, these are the names that core will add in 4.20, so we'll have to remove them to keep compiling. I think that's enough of a reminder :)